includes/User.php

```diff
@@ -4894,9 +4894,8 @@ private function loadAttributes() {
 			foreach ( $attributes as $attributeName => $attributeValue ) {
 				$this->compareAttributeValueFromService( $attributeName, $attributeValue );
```

Comment:
Shouldn't we raise an exception here (when there's a mismatch) instead of writing wrong data?